src/software/ai/hl/stp/tactic/move_tactic.h

```diff
@@ -1,7 +1,55 @@
 #pragma once
 
-#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/action/move_action.h"  // TODO (#1888): remove this dependency
 #include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/ai/intent/move_intent.h"
+
+struct MoveFSM
+{
+    struct Move
```

Comment:
It would be good to have comment lines to separate state structs from other structs.

Reply:
done